src/content/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure.mdx

```diff
@@ -32,54 +32,33 @@ import kubernetesCke from 'images/kubernetes_logo_cke.png'
+The New Relic Kubernetes integration gives you full observability into the health and performance of your environment by leveraging the New Relic infrastructure agent. This agent collects metrics from your cluster alongside various other New Relic integrations such as the [Kubernetes events integration](/docs/integrations/kubernetes-integration/kubernetes-events/install-kubernetes-events-integration), the [Prometheus Agent](/docs/infrastructure/prometheus-integrations/install-configure-prometheus-agent/install-prometheus-agent/), and the [New Relic log plugin](/docs/logs).
 
-As a Kubernetes user, you might consider [Auto-telemetry with Pixie](/docs/auto-telemetry-pixie/get-started-auto-telemetry-pixie) for deeper insights into your Kubernetes clusters and workloads. No language agents required.
-
-Ready to get started? If you don't already have one, [sign up for a New Relic account](https://newrelic.com/signup). It's free, forever!
+You have these options to install the Kubernetes integration:
 
-## Use guided install [#installer]
+* The [New Relic Kubernetes quickstart](https://newrelic.com/instant-observability/kubernetes). This quickstart includes pre-built resources like dashboards, instrumentations, and alerts.
 
-We encourage you to use our guided install for servers, VMs, and unprivileged environments. The guided install can provide you either a Helm command with the required values filled, or a plain manifest if you don't want to use Helm. It also features great customizability and full control over which features and dependencies are enabled. If you are installing our integration on a managed cloud, please take a look at these [preliminary notes](#cloud-platforms) before proceeding.
+* The [automated and guided install](https://one.newrelic.com/launcher/k8s-cluster-explorer-nerdlet.cluster-explorer-launcher?pane=eyJuZXJkbGV0SWQiOiJucjEtaW5zdGFsbC1uZXdyZWxpYy5ucjEtaW5zdGFsbC1uZXdyZWxpYyIsInBhdGgiOiJndWlkZWQiLCJlbnYiOiJrdWJlcm5ldGVzIiwiaW5pdGlhbEFjdGlvbkluZGV4IjpudWxsLCJhY3Rpb25JbmRleCI6MX0=) flow. If you follow the Kubernetes guided install docs below you've these options to install the Kubernetes integration:
+  * New Relic CLI
+  * [Helm](https://onenr.io/0Y8wpoYJJQO). See [Install the Kubernetes integration using Helm](/docs/kubernetes-pixie/kubernetes-integration/installation/install-kubernetes-integration-using-helm)
+  * Manifest
 
-Before you start the installer, make sure you have a New Relic account. It's free and no credit card is required.
-
-If your New Relic account [is in the EU region](/docs/using-new-relic/welcome-new-relic/get-started/our-eu-us-region-data-centers), choose the EU install option.
+As a Kubernetes user, you might consider [Auto-telemetry with Pixie](/docs/auto-telemetry-pixie/get-started-auto-telemetry-pixie) for deeper insights into your Kubernetes clusters and workloads. No language agents required.
```

Comment:
@nbaenam this is the first time I've seen anything that differentiates the Kubernetes integration from the Pixie integration. Is there more information like this somewhere? As a user, do I need to get into the installation instructions before I understand the difference between these two products?